src/fsharp/absil/ilwritepdb.fs

```diff
@@ -685,7 +685,7 @@ type PortablePdbGenerator (embedAllSource: bool, embedSourceList: string list, s
         metadata.SetCapacity(TableIndex.MethodDebugInformation, info.Methods.Length)
 
 // Currently disabled, see 
-#if EMIT_IMPORT_SCOPES
+#if !NO_EMIT_IMPORT_SCOPES
```

Comment:
did we set EMIT_IMPORT_SCOPES, do we set NO_EMIT_IMPORT_SCOPES?

do we really need the ````#if````

Reply:
Not really, it can be remote really